dask/tests/test_distributed.py

```diff
@@ -659,3 +659,27 @@ async def test_pack_MaterializedLayer_handles_futures_in_graph_properly(c, s, a,
     packed = hlg.__dask_distributed_pack__(c, ["z"], {})
     unpacked = HighLevelGraph.__dask_distributed_unpack__(packed)
     assert unpacked["deps"] == {"x": {fut.key}, "y": {fut.key}, "z": {"y"}}
+
+
+@gen_cluster(client=True)
+async def test_to_sql_engine_kwargs(c, s, a, b):
+    # https://github.com/dask/dask/issues/8738
+    pd = pytest.importorskip("pandas")
+    dd = pytest.importorskip("dask.dataframe")
+    sql = pytest.importorskip("dask.dataframe.io.sql")
+
+    df = pd.DataFrame({"a": range(10), "b": range(10)})
+    ddf = dd.from_pandas(df, npartitions=1)
+    with tmpfile() as f:
+        uri = f"sqlite:///{f}"
+        result = ddf.to_sql(
+            "test", uri, index=True, engine_kwargs={"echo": False}, compute=False
+        )
+        await c.compute(result)
+
+        dd.utils.assert_eq(
+            ddf,
+            sql.read_sql_table("test", uri, "index"),
```

Comment:
Nitpick: I think using `dd` here will be less surprising to future readers

```suggestion
            dd.read_sql_table("test", uri, "index"),
```

Reply:
:facepalm: forgot it was available from there.